components/organisms/o-order-summary.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+  <div class="o-order-summary">
+    <div class="highlighted">
+      <SfHeading
+        :title="$t('Order Summary')"
+        class="sf-heading--left sf-heading--no-underline title"
+      />
+      <div class="total-items">
+        <h3>{{ $t("Total items") }}: {{ totalItems }}</h3>
+        <SfButton
+          class="sf-button--text"
+          @click="showProducts = !showProducts"
+        >
+          {{ showProducts ? $t("Hide products") : $t("Show products") }}
+        </SfButton>
+      </div>
+      <transition name="fade">
+        <div v-if="showProducts" class="collected-product-list">
+          <SfCollectedProduct
+            v-for="product in productsInCart"
+            :key="product.id"
+            class="collected-product"
+            :image="getThumbnailForProduct(product)"
+            :title="product.name | htmlDecode"
+            :regular-price="getProductRegularPrice(product)"
+            :special-price="getProductSpecialPrice(product)"
+            :stock="product.stock.qty"
+            :qty="product.qty"
+            @click:remove="removeProduct(product)"
+            @input="changeQuantity(product, $event)"
+          >
+            <template #configuration>
+              <div class="product__properties">
+                <SfProperty
+                  v-for="property in product.options"
+                  :key="property.label"
+                  :name="property.label"
+                  :value="property.value"
+                  class="product__property"
+                />
+              </div>
+            </template>
+            <template #actions>
+              <div>
+                <div class="product__action">
+                  {{ product.sku }}
+                </div>
+                <div class="product__action">
+                  {{ $t("Quantity") }}:
+                  <span class="product__qty">{{ product.qty }}</span>
+                </div>
+              </div>
+            </template>
+          </SfCollectedProduct>
+        </div>
+      </transition>
+    </div>
+    <MPriceSummary class="highlighted highlighted--total" />
+    <div class="highlighted promo-code">
```

Comment:
Can we make here atom or molecule? a-promo-code 
Then we can easy add promo code in different organism

Reply:
Done: `a-promo-code` has been added.